@xen-orchestra/xapi/vm.mjs

```diff
@@ -541,6 +541,10 @@ class Vm {
     return Object.keys(disks)
   }
 
+  async getSecurebootReadiness(vmRef) {
+    return this.call('VM.get_secureboot_readiness', vmRef)
+  }
```

Comment:
Don't create the method in `@xen-orchestra/xapi/*.mjs` if that simply a wrapper.